ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils_filtered_tenants.py

```diff
@@ -52,10 +52,32 @@ def _filtered_l2vlans(self, tenant: dict) -> list[dict]:
         l2vlans = [
             l2vlan
             for l2vlan in l2vlans
-            if (int(l2vlan["id"]) in self._accepted_vlans and ("all" in self._filter_tags or set(l2vlan.get("tags", ["all"])).intersection(self._filter_tags)))
+            if self._is_accepted_vlan(l2vlan) and ("all" in self._filter_tags or set(l2vlan.get("tags", ["all"])).intersection(self._filter_tags))
         ]
         return l2vlans
 
+    def _is_accepted_vlan(self, vlan: dict) -> bool:
+        """
+        Check if vlan is in accepted_vlans list
+        If filter.only_vlans_in_use also check if vlan id or trunk group is assigned to connected endpoint
```

Comment:
```suggestion
        If filter.only_vlans_in_use is True also check if vlan id or trunk group is assigned to connected endpoint
```

Makes the sentence easier to read